src/runtime/task/mod.rs

```diff
@@ -38,6 +41,74 @@ use waker::make_waker;
 
 pub(crate) const DEFAULT_INLINE_TASKS: usize = 16;
 
+/// To make debugging easier, if a task is assigned a `TaskName(s)` Label,
+/// Shuttle will display the String `s` in addition to the `TaskId` in debug output.
```

Comment:
nit: same thing about docs style being one-line summary of "what the thing is", then more paragraphs

Reply:
Done